docs/Installation.md

```diff
@@ -148,15 +148,43 @@ Now to know how to update the `src/config/supported_distros.py` to reflect the n
 Step 2 of
 [Adding_new_distros](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Adding_new_distros.md#step-2-update-the-supported_distros-variable-in-configuration-file-sdt_basesrcconfigconfigpy)
 
-###  Step 6: Verify that the software-discovery-tool server is up and running
-* For Ubuntu (18.04, 19.04, 20.04, 22.04):
+### Step 6: Install and populate the SQL database
+* For Ubuntu (18.04, 20.04, 22.04):
+
+#### Install dependencies
+
+        sudo apt install mariadb-server python3-pymysql
+
+#### Run MariaDB from the command line and complete the secure installation. Remember the root password you set, you will need this in the future.
+
+        sudo mariadb
+        sudo mysql_secure_installation
+
+#### Log in to MariaDB with the root account you set and create the read-only user (with a password, changed below) and database.
+
+        mariadb -u root -p
+        MariaDB> GRANT SELECT ON sdtDB.* TO 'sdtreaduser'@localhost IDENTIFIED BY 'CHANGE_ME';
```

Comment:
write access should also be there for the database..I believe the current tool allows us to write to tables in `sdtDB`

Reply:
more like this `GRANT ALL PRIVILEGES ON sdtDB.* TO 'sdtreaduser'@localhost IDENTIFIED BY 'CHANGE_ME';`

Reply:
The only thing that uses this access is package_search.py, and we don't want that to have write access to the database, since it's being executed by users searching via the web.

Any changes made to the database are done via database_build.py and that asks for an administrative password when you run it (I've just been running it with the root mysql password). Though we could suggest setting up a separate user that has GRANT ALL, but that would be separate from this one.

Reply:
@arshPratap This is how I'm running it in production.

Think we should move forward with the instructions as-is, or should we advise the setup of an administrative user as well?